zerver/tests/test_export.py

```diff
@@ -239,15 +256,80 @@ def test_attachment_and_emoji(self) -> None:
         record = data['zerver_attachment'][0]
         self.assertEqual(record['path_id'], path_id)
 
+        # Test uploads
         fn = os.path.join(full_data['uploads_dir'], path_id)
         with open(fn) as f:
             self.assertEqual(f.read(), 'zulip!')
 
+        # Test emojis
         fn = os.path.join(full_data['emoji_dir'],
                           RealmEmoji.PATH_ID_TEMPLATE.format(realm_id=realm.id, emoji_file_name='1.png'))
         fn = fn.replace('1.png', '')
         self.assertEqual('1.png', os.listdir(fn)[0])
 
+        # Test avatars
+        fn = os.path.join(full_data['avatar_dir'], original_avatar_path_id)
+        fn_data = open(fn, 'rb').read()
+        self.assertEqual(fn_data, test_image)
+
+        message.sender.avatar_source = 'G'
+        message.sender.save()
```

Comment:
You don't need to do this; the database is automatically rolled back to the pre-test state at the end of a test.  You only need to worry about this sort of thing with the filesystem, since we don't have something like database transactions for doing automatic rollback there :)